user_role_analysis/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 ForgeFlow, S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "User Role Analysis",
+    "version": "15.0.1.0.0",
+    "summary": "specific user role analysis reports",
```

Comment:
can you elaborate a bit more here? Also, having a readme fragment + screenshot of what you get as result would be nice :pray: